veps/sig-compute/image-volume-proposal.md

```diff
@@ -0,0 +1,169 @@
+# VEP #12: Your short, descriptive title
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
```

Comment:
You can mark this as done since you've opened an issue already
```suggestion
- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
```

Reply:
Done